### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by ryanfortner**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,20 +1,33 @@
...
+<p align="center">Linux Userspace x86_64 Emulator with a twist
+<p align="center">
+  <a href="https://github.com/ptitSeb/box64/blob/main/docs/CHANGELOG.md">
+    View changelog</a>
+  |
+  <a href="https://github.com/ptitSeb/box64/blob/main/docs/README_CN.md">
+    中文</a>
+  |
+  <a href="https://github.com/ptitSeb/box64/issues/new">
+    Report an error</a>
```
Comment (ptitSeb) [inline docs/README.md:12]: I don't like having html code inside the makdown file. I prefer to keep markdown syntax only.

**Review by ptitSeb — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 1 above)
Comment (ryanfortner) [reply docs/README.md]: Hmm... is there any way to center text in markdown without using html?

Comment (Itai-Nelken) [reply docs/README.md:12]: markdown doesn't support aligning. github markdown supports with HTML.

Comment (ptitSeb) [reply docs/README.md:12]: Yes, but I don't like poluting markdown file with html. Please don't try to align.

**Comment by ryanfortner** [comment]
@ptitSeb is this better?

**Review by ptitSeb — APPROVED** [review]
Thanks :)

**Pull request merged**